package.json

```diff
@@ -58,14 +58,14 @@
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.5",
-    "@tsconfig/node16": "^1.0.3",
+    "@tsconfig/node16": "^16.1.1",
```

Comment:
I've decided to not upgrade our tsconfig to `@tsconfig/node18` because that one uses `es2023` as a `lib`, which our current Typescript version doesn't support. I also don't want to risk any unexpected JS code changes in our bundle output at this point.